### PR TITLE
Feature/action tests

### PR DESCRIPTION
**Pull request by jenniferlynparsons**

### CODE REVIEW

File: client/src/actions/__tests__/authActions.test.js [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+  length: 10,
+  action: "PUSH",
+  location: { pathname: "/dashboard", search: "", hash: "", key: "zh4boo" },
+  push: jest.fn()
+};
+
+const mockResponse = {
+  success: true,
+  token:
+    "Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IjVjNjMxM2E0YzMxOGJiNjIyOThiMjNkNCIsIm5hbWUiOiJKZW5uaWZlciIsImlhdCI6MTU1ODk3MTY4MSwiZXhwIjoxNTkwNTI4NjA3fQ.1Tn8FLJE2gUj0D0ka91e79cLEQz6ttSkQ0sZNd3GGE8"
```
Comment (JeffreyATW) [inline client/src/actions/__tests__/authActions.test.js:37]: Just checking, is this a real auth token? Hopefully you're not checking in possibly sensitive information.

File: client/src/actions/__tests__/authActions.test.js [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+  test("it resets the authToken", async () => {
+    await store.dispatch(logoutUser());
+    expect(setAuthToken).toHaveBeenCalled();
+  });
+});
+
+describe("setCurrentUser", () => {
+  test("returns a function", () => {
+    expect(setCurrentUser(mockDecodedToken)).toBeInstanceOf(Object);
+  });
```
Comment (JeffreyATW) [inline client/src/actions/__tests__/authActions.test.js:138]: Looks like the description of this test is wrong.

File: client/src/actions/__tests__/flashActions.test.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+const store = makeMockStore({});
+const mockStatus = "on";
+
+describe("editTeaFlash", () => {
+  beforeEach(() => {
+    store.clearActions();
+  });
+  test("returns a function", () => {
+    expect(editTeaFlash(mockStatus)).toBeInstanceOf(Object);
+  });
```
Comment (JeffreyATW) [inline client/src/actions/__tests__/flashActions.test.js:13]: Looks like the description of this test is wrong.

File: client/src/actions/__tests__/teaActions.test.js [hunk 4]
```diff
@@ -0,0 +1,202 @@
...
+  test("it returns the ADD_TEA action type and payload", async () => {
+    await store.dispatch(addTea(mockAddTea));
+    expect(store.getActions()[0].payload).toEqual(mockPostResponse);
+    expect(store.getActions()[0].type).toEqual("ADD_TEA");
+  });
+});
+
+describe("editTea", () => {
+  beforeEach(() => {
+    store.clearActions();
```
Comment (JeffreyATW) [inline client/src/actions/__tests__/teaActions.test.js]: It's probably easier to just re-create the store in a `beforeEach` block at the top level of this file.

File: client/src/actions/teaActions.js [hunk 5]
```diff
@@ -2,33 +2,34 @@ import API from "../lib/api";
 import { teaActionTypes } from "../lib/actionTypes";
 
 // Add Tea
-export function addTea(tea) {
+export const addTea = tea => {
+  debugger;
```
Comment (JeffreyATW) [inline client/src/actions/teaActions.js]: Remove debugger statement

**Review by JeffreyATW — CHANGES_REQUESTED** [review]

File: client/src/actions/__tests__/authActions.test.js (hunk 1 above)
Comment (jenniferlynparsons) [reply client/src/actions/__tests__/authActions.test.js:37]: it was real, but it's for a garbage account that's been deleted so no longer exists.

**Review by JeffreyATW — APPROVED** [review]

**Pull request merged**